e2e-tests/development-runtime/package.json

```diff
@@ -56,5 +56,8 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/gatsbyjs/gatsby"
+  },
+  "resolutions": {
+    "cypress": "3.4.1"
```

Comment:
Attempt to fix https://app.circleci.com/pipelines/github/gatsbyjs/gatsby/48158/workflows/3227d59e-b7ae-4eeb-b109-06a9712b44c2/jobs/491859

Some existing tickets about this problem:
 - https://github.com/testing-library/cypress-testing-library/issues/98
 - https://github.com/cypress-io/cypress/issues/7951
 - https://github.com/cypress-io/cypress/issues/4595